.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Workflow
+
+permissions:
+  id-token: write
+  contents: read
```

Comment:
what are these top level permissions for?

Reply:
I think this is needed for the tagging